security/backends/elasticsearch/tests.py

```diff
@@ -21,7 +22,11 @@ def enable(self):
         for document_class in (CommandLog, CeleryTaskRunLog, CeleryTaskInvocationLog,
                                InputRequestLog, OutputRequestLog):
             document_class._index._name = f'{uuid}.{document_class._index._name}'
-            document_class.init()
+            if issubclass(document_class, PartitionedLog):
+                template = document_class.get_template()
+                template.save()
+            else:
+                document_class.init()
```

Comment:
Wouldn't it be better to override the `init` method of the `PartitionedLog` and save the template there? You could remove this whole if/else change in both places and it would comform to the pattern of the other Documents.

Reply:
`init` is particularly for creating index, while I create here just a template, so it might be confusing imho. This implementation is directly from documentation of `elasticsearch-dsl` for time-based indexes.
But it could be in some function, that's for sure.